generate.sh

```diff
@@ -13,7 +13,7 @@ FILE="wordpress-stubs.php"
 
 # Shim the global $wpdb declaration, since it's actually set up inside a
 # function call.
-echo $'\n/**\n * WordPress database abstraction object.\n * @var wpdb\n */\n$wpdb = null;' >> $FILE
+echo $'\nnamespace {\n\t/**\n\t * WordPress database abstraction object.\n\t * @var wpdb\n\t */\n\t$wpdb = \\null;\n}' >> $FILE
```

Comment:
The `$wpdb` shim needs to be wrapped in a global namespace declaration as of WP 5.2.0, due to the minimum PHP version bump up to 5.6.20 and the introduction of namespaces.